flow/accounts/create/create.go

```diff
@@ -37,6 +39,7 @@ type Config struct {
 	HashAlgo string   `default:"SHA3_256" flag:"hash-algo" info:"Hash used for the digest"`
 	Host     string   `flag:"host" info:"Flow Access API host address"`
 	Results  bool     `default:"false" flag:"results" info:"Display the results of the transaction"`
+	Contract string   `flag:"contract,c" info:"Contract to be deployed during account creation. <name:path>"`
```

Comment:
Good idea to use `name:path`. Maybe make it a slice like `Keys` above?

Reply:
that's a pretty easy way to make it support multiple contracts, so sure! added.